spec/rails_helper.rb

```diff
@@ -95,22 +95,36 @@
     DatabaseCleaner.clean
   end
 
+  def stub_oauth_connection
```

Comment:
I don't think these methods should be in rails_helper. I missed the PR when this first got put in this file. I'd rather see them in a subdirectory file like `spec/helpers/`.